lib/core/domain/collaborator/collaborator_repository.dart

```diff
@@ -24,4 +26,8 @@ abstract class CollaboratorRepository {
   Future<Either<Failure, List<UserDiscoverySwipe>>> getUserDiscoverySwipes({
     required Variables$Query$GetUserDiscoverySwipes input,
   });
+
+  Future<Either<Failure, List<UserExpertise>>> getListUserExpertises();
```

Comment:
these should be in UserRepository, as it relates to User, not collaborator, 
same as getListUserServices